src/App.tsx

```diff
@@ -1,46 +1,43 @@
+import { RouterProvider, createBrowserRouter, json } from "react-router-dom";
+import "./App.css";
+import AuthLayout from "./SharedModule/components/AuthLayout/AuthLayout";
+import NotFound from "./SharedModule/components/NotFound/NotFound";
+import Login from "./AuthModule/components/Login/Login";
+import Register from "./AuthModule/components/Register/Register";
+import ForgotPassword from "./AuthModule/components/ForgotPassword/ForgotPassword";
+import ReasetPassword from "./AuthModule/components/ReasetPassword/ReasetPassword";
+import VerifyAccount from "./AuthModule/components/VerifyAccount/VerifyAccount";
+import MasterLayout from "./SharedModule/components/MasterLayout/MasterLayout";
+import Dashboard from "./Dashboard/components/Dashboard/Dashboard";
+import ProjectList from "./ProjectModule/components/ProjectList/ProjectList";
+import UserList from "./UsersModule/Components/userList/UserList";
+import TasksList from "./TasksModule/components/TasksList/TasksList";
 
-import { RouterProvider, createBrowserRouter } from 'react-router-dom';
-import './App.css'
-import AuthLayout from './SharedModule/components/AuthLayout/AuthLayout';
-import NotFound from './SharedModule/components/NotFound/NotFound';
-import Login from './AuthModule/components/Login/Login';
-import Register from './AuthModule/components/Register/Register';
-import ForgotPassword from './AuthModule/components/ForgotPassword/ForgotPassword';
-import ReasetPassword from './AuthModule/components/ReasetPassword/ReasetPassword';
-import VerifyAccount from './AuthModule/components/VerifyAccount/VerifyAccount';
-import MasterLayout from './SharedModule/components/MasterLayout/MasterLayout';
+import 'react-toastify/dist/ReactToastify.css';
 
-import Dashboard from './Dashboard/components/Dashboard/Dashboard';
-import ProjectList from './ProjectModule/components/ProjectList/ProjectList';
-import UserList from './UsersModule/Components/userList/UserList';
-import TasksList from './TasksModule/components/TasksList/TasksList';
+import ProtectedRoute from "./SharedModule/components/ProtectedRoute/ProtectedRoute";
+import { useContext } from "react";
+import { AuthContext } from "./context/AuthContext";
 function App() {
 
-
-
-
-
+ const { loginData,savLoginData} = useContext(AuthContext)
   const router = createBrowserRouter([
```

Comment:
The convention with naming URL paths is to make it small letters, and when having multiple words to separate  them with hyphen as the example below from Next.js page. 
![image](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/cc3e8bb4-ff7e-46d0-89c4-acdc929f1891)

The problem with using camelCase syntax is that search engines will treat these paths as a single word. If we try to select  it, we will find the following:
![URL Naming](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/e17538c9-a6a7-4d60-910a-077cf5648225)